Sources/StickeyCollectionViewLayout/StickyCollectionViewLayout.swift

```diff
@@ -0,0 +1,178 @@
+                self.itemsSize.append(self.sizeForItemWithColumnIndex(index))
+            }
+        }
+    }
+
+    private func stickColumnHeaderAndFirstColumn() {
+        guard let collectionView = collectionView,
+            let numberOfSections = self.collectionView?.numberOfSections,
+            numberOfSections > 0 else {
+                return
+        }
+
+        let contentOffset = collectionView.contentOffset
+        for section in 0..<numberOfSections {
+            let numberOfItems : Int = collectionView.numberOfItems(inSection: section)
+            for index in 0..<numberOfItems {
+                if section != 0 && index != 0 {
+                    continue
+                }
+
+                let attributes = self.layoutAttributesForItem(at: IndexPath(item: index, section: section))!
+
+                // Stick the first row
+                if section == 0 {
+                    var frame = attributes.frame
+                    frame.origin.y = contentOffset.y
+                    attributes.frame = frame
+                }
+
+                // Stick the first column
+                if index == 0 {
+                    var frame = attributes.frame
+                    frame.origin.x = contentOffset.x
+                    attributes.frame = frame
+                }
+            }
+        }
+    }
+
+    private func calculateAllItemAtrributes() {
+
+        guard let collectionView = collectionView,
+            let numberOfSections = self.collectionView?.numberOfSections,
+            numberOfSections > 0 else {
+                return
+        }
+
+        self.calculateItemsSize()
+
+        var column = 0
+        var xOffset : CGFloat = 0
+        var yOffset : CGFloat = 0
+        var contentWidth : CGFloat = 0
+        var contentHeight : CGFloat = 0
+
+        for section in 0..<self.collectionView!.numberOfSections {
+            var rowAttributes = [UICollectionViewLayoutAttributes]()
+            let numberOfColumns = collectionView.numberOfItems(inSection: section)
+            let contentOffset = collectionView.contentOffset
+
+            for index in 0..<numberOfColumns {
+                let itemSize = (self.itemsSize[index] as AnyObject).cgSizeValue
+                let indexPath = IndexPath(item: index, section: section)
+                let rowAttribute = UICollectionViewLayoutAttributes(forCellWith: indexPath)
+                rowAttribute.frame = CGRect(x: xOffset, y: yOffset, width: (itemSize?.width)!, height: (itemSize?.height)!).integral
+
+                if section == 0 && index == 0 {
+                    rowAttribute.zIndex = 1024;
+                } else  if section == 0 || index == 0 {
+                    rowAttribute.zIndex = 1023
+                }
+
+                // Stick the first row
+                if section == 0 {
+                    var frame = rowAttribute.frame
+                    frame.origin.y = contentOffset.y
+                    rowAttribute.frame = frame
```

Comment:
I think you can directly modify the `y` value like this `attributes.frame.origin.y = contentOffset.y`